### ISSUE TITLE
Debloat CSS

### ISSUE THREAD

**Issue by baltpeter** [datenanfragen/website#596]
While it isn't the end of the world at currently 23KB, our CSS could definitely do with a bit of optimizing. Two things in particular I can think of right now:

* The `bg-*` and `color-*` utilities. Apart from `/design`, they are barely used and take up a lot of space in relation to their use. I'm not sure what the best course of action is here but this seems inefficient.  
  Maybe we want to hardcode the values on the design page (which we are already doing for the text anyway) and then only include the classes that we actually use? Not sure.
* There's quite a bit of stuff (mainly in `generator.scss` and `main.scss`) that is only needed in a few places and doesn't have to be included everywhere.

**Comment by Divyateja04** [comment]
Hey, I would like to work on this, can you assign this to me?

**Comment by Divyateja04** [comment]
Can you also give more details though if you don't mind?

**Comment by baltpeter** [comment]
@Divyateja04 This isn't really an issue that I can give you more details on as the purpose is to research that. You would need to either be familiar with our code base already or willing to dig in yourself.

For more guided issues, take a look at the ones tagged [good first issue](https://github.com/datenanfragen/website/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22).

**Comment by Divyateja04** [comment]
Yeah but most of them are assigned and remaining are translation ones and I don't know most of the languages which would mess it up again so is there any issue I can actually work on?

**Comment by baltpeter** [comment]
Sorry, nothing we can do about. We'll add new issues as we think of them.

Otherwise, you can always try and use the site to find bugs or things you can improve yourself.

**Comment by Divyateja04** [comment]
yeah I found another issue infact, see if I can work on it

**Comment by encodeArnab** [comment]
Hello @baltpeter ,
I'd want to assist with this issue; I've reviewed the codebase and am confident that I can assist with the optimization of ```generator.scss``` and ```main.scss```.
Can you assign it to me.
Thank you

**Comment by baltpeter** [comment]
Thanks @encodeArnab, I have assigned you.

**Comment by encodeArnab** [comment]
Hello @baltpeter,
I've tried running Hugo server several times, but it never returned this result, and the server isn't even running. I've followed all of the instructions. 
It says in the output ```fatal error: pipe failed```
What am I supposed to do?

**Comment by baltpeter** [comment]
@encodeArnab That sounds like an error with Hugo or your Hugo installation. Unfortunately, I can't help you with that.
